### DISCUSSION TITLE
[Bug]: minValue not work

### DISCUSSION THREAD

**Discussion by mauriciomgp5** [leandrocfe/filament-ptbr-form-fields#56] [Q&A]
### What happened?

Validation minValue, not work

### How to reproduce the bug

PtbrMoney::make('amount')
                ->label('Valor')
                ->minValue(1) //this not work, or 0.01
                ->required(),
Forms\Components\TextInput::make('quantity')
            ->label('Quantidade')
            ->numeric()
            ->minValue(1)
            ->required(),

### Package Version

2x

### PHP Version

8.1

### Laravel Version

10

**Comment by jlucfarias** [comment]
I get it working using this because minValue rule consider a simple text, not numeric. @leandrocfe could you please add it to the field?

```diff
PtbrMoney::make('amount')
    ->label('Valor')
+   ->mutateStateForValidationUsing(fn ($state) => Str::of($state)->replace(',', '')->toInteger() / 100)
+   ->rule('numeric')
    ->minValue(1)
    ->required(),
```
